### PR TITLE
Backend top articles

### PR DESCRIPTION
**Pull request by memhak**
Implemented get top articles feature

### CODE REVIEW

File: backend/src/main/java/com/eyetrade/backend/service/ArticleService.java [hunk 1]
```diff
@@ -31,6 +31,16 @@ public ArticlesResource getArticles(){
         return  new ArticlesResource(resources);
     }
 
+    public ArticlesResource getTopArticles(){
+        List<ArticleResource> resources = new ArrayList<ArticleResource>();
+        List<Article> articles = articleRepository.findAllByOrderByScoreDesc();
+
+        for(Article article: articles){
```
Comment (gokdeemir) [inline backend/src/main/java/com/eyetrade/backend/service/ArticleService.java]: We can add null check such as collectionutils.emptyifnull(<list>)

File: backend/src/main/java/com/eyetrade/backend/service/ArticleService.java [hunk 2]
```diff
@@ -41,6 +51,16 @@ public ArticlesResource getArticles(String userEmail){
         return new ArticlesResource(resources);
     }
 
+    public ArticlesResource getTopArticles(String userEmail){
+        List<ArticleResource> resources = new ArrayList<ArticleResource>();
+        List<Article> articles = articleRepository.findAllByAuthorEmailOrderByScoreDesc(userEmail);
+
+        for (Article article: articles){
```
Comment (gokdeemir) [inline backend/src/main/java/com/eyetrade/backend/service/ArticleService.java]: Null check again

File: backend/src/main/java/com/eyetrade/backend/service/ArticleService.java (hunk 1 above)
Comment (memhak) [reply backend/src/main/java/com/eyetrade/backend/service/ArticleService.java]: In this case, it returns blank list, not null. Do we still need null check? Blank list would imply no articles.

Comment (gokdeemir) [reply backend/src/main/java/com/eyetrade/backend/service/ArticleService.java]: If that way, it's okay

**Pull request merged**